src/gui/editorwidgets/qgsrelationreferencewidgetwrapper.cpp

```diff
@@ -218,6 +220,9 @@ void QgsRelationReferenceWidgetWrapper::setEnabled( bool enabled )
 
 void QgsRelationReferenceWidgetWrapper::foreignKeysChanged( const QVariantList &values )
 {
+  if ( mBlockChanges ) // initial value is being set, we can ignore this signal
```

Comment:
```suggestion
  if ( mBlockChanges != 0  ) // initial value is being set, we can ignore this signal
```

just for clarity to remind the reader the it isn't a bool